src/helpers/authValidation.js

```diff
@@ -0,0 +1,63 @@
+const authValidation = (fieldName, formInput) => {
```

Comment:
For consistent form validations across the app, could you take a look at Formik. Here's a link from the ReactJS docs which gives an overview of the library - https://reactjs.org/docs/forms.html#fully-fledged-solutions

Reply:
okay.. would check it out now